src/twisted/web/http.py

```diff
@@ -1910,6 +1917,9 @@ def __init__(
         self.finishCallback = finishCallback
         self._buffer = bytearray()
         self._start = 0
+        self._trailerHeaders: List[bytearray] = []
+        self._maxTrailerHeadersSize = 16384
```

Comment:
See https://github.com/twisted/twisted/issues/8570#issuecomment-1864682301

No need to update the value in this PR.

But I think that it would help to define this value as a private constant for now.

Later we can look at sharing the same constant across multiple places.

maybe something like this ?

```suggestion
        self._maxTrailerHeadersSize = 2 ** 16
```